src/lightning/app/cli/react-ui-template/ui/yarn.lock

```diff
@@ -0,0 +1,1138 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@Borda should this file be here? If this is auto generated it probably shouldn't!

Reply:
I would like to remove it #16576, but it needs some dependency fix as the UI fails without it :/

Reply:
see source: https://github.com/Lightning-AI/lightning/blob/3001c7c1d5881ff917af17662582b1735b1cfcaa/src/lightning_app/cli/react-ui-template/ui/yarn.lock